### DISCUSSION TITLE
Tempo-match 2 singers

### DISCUSSION THREAD

**Discussion by sdflemming** [Anjok07/ultimatevocalremovergui#1273] [Ideas]
I would like to have the ability to take two audio file recordings of two different singers singing the same song but out of tempo and tempo-match the singers so their tempos and "words" match. Perhaps the ability to tweak the pitch would also be good to pitch-match singers.

**Comment by gue88** [comment]
you need to do this in a DAW

**Comment by Gravitationalwavesarecool** [comment]
I use audacity for altering tempo, I use vocal remover so I can use just the vocals from both into audacity, then you can load both versions in and only select the one you want to change and you can use their tempo button to match the Soundwaves together
